subprojects/test-runner/device-provider/src/main/kotlin/com/avito/android/runner/devices/internal/AbstractDevice.kt

```diff
@@ -25,13 +28,15 @@ internal abstract class AbstractDevice(protected val loggerFactory: LoggerFactor
             " ${tags.joinToString(" ")}"
         }
 
-        executeNonBlockingCommand(
-            command = "logcat$tagsString",
-            redirectOutputTo = file
+        // TODO stop endless process
```

Comment:
Bug: we don't how this process could load our builder when we free pod connected with that process